poetry.lock

```diff
@@ -460,20 +460,20 @@ dev = ["pre-commit", "tox"]
 
 [[package]]
 name = "pre-commit"
-version = "2.11.0"
+version = "2.11.1"
```

Comment:
[pre-commit 2.11.1](https://github.com/pre-commit/pre-commit/blob/master/CHANGELOG.md#2111---2021-03-09) is a bugfix